pages/common/pyinfra.md

```diff
@@ -0,0 +1,28 @@
+# pyinfra
+
+> Automates infrastructure at a large scale.
+> More information: <https://docs.pyinfra.com>.
+
+- Execute a command over ssh:
```

Comment:
```suggestion
- Execute a command over `ssh`:
```

Reply:
```suggestion
- Execute a command over `SSH`:
```

Reply:
This doesn't make any sense to have SSH in backticks, because SSH is the name of a protocol and not a command.